static/js/modules/game/pagePreview.js

```diff
@@ -0,0 +1,42 @@
+var PagePreview = {
+
+	props: {
+		articlePreview: Object,
+		clientX: Number,
+		clientY: Number
+	},
+
+	methods: {
+		computePosition: function() {
+            const vh = window.innerHeight;
+            const vw = window.innerWidth;
+            const styleObject = {};
+
+            if (vw >= 768) {
+                if (this.clientX < vw / 2.0) {
+                    styleObject['left'] = `${this.clientX+10}px`;
+                } else {
+                    styleObject['right'] = `${vw-this.clientX+10}px`;
+                }
+                if (this.clientY < vh / 2.0) {
+                    styleObject['top'] = `${this.clientY+10}px`;
+                } else {
+                    styleObject['bottom'] = `${vh-this.clientY+10}px`;
+                }
+            } else {
+                styleObject['left'] = `${Math.floor((vw-360)/2)}px`;
+                styleObject['bottom'] = `${vh-this.clientY+25}px`;
+            }
+            return styleObject;
+        }
+	},
+
+	template: (`
+	<div class="tooltip-container" :style="computePosition()">
+        <img v-if="articlePreview.originalimage" :src="articlePreview.originalimage.source" />
```

Comment:
Instead of 'original-image', can we use 'thumbnail' instead? the original image is often a couple times larger than thumbnail, and takes considerably longer to load

See: https://en.wikipedia.org/api/rest_v1/page/summary/United_States as an exmaple

Reply:
I can make this change